### PR TITLE
feat: Added alias lookup

### PR DESCRIPTION
**Pull request by kiesman99**
Alias lookup functionality highly inspired by [zimfw/git](https://github.com/zimfw/git/blob/master/functions/git-alias-lookup)

### CODE REVIEW

File: init.zsh [hunk 1]
```diff
@@ -118,4 +118,7 @@
   for r in ${(k)resources}; do
     alias "${kprefix}c${r}"="kubectl create ${resources[$r]}"
   done
+
+  # misc
+  alias ${kprefix}\?="kubectl-alias-lookup $1"
```
Comment (joke) [inline init.zsh:123]:
```suggestion
  alias "${kprefix}?=kubectl-alias-lookup $1"
```

Comment (joke) [reply init.zsh:123]: A little bit easier to read.

**Review by joke — CHANGES_REQUESTED** [review]

File: functions/kubectl-alias-lookup [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/bin/zsh
+if (( # < 1 )); then
+  print -u2 "usage: ${0} <path to git module> [pattern...]"
```
Comment (joke) [inline functions/kubectl-alias-lookup:3]:
```suggestion
  print -u2 "usage: ${0} <path to kubectl module> [pattern...]"
```

**Comment by kiesman99** [comment]
The [original Version](https://github.com/zimfw/git/blob/master/functions/git-alias-lookup) includes the descriptions provided in the [README](https://github.com/zimfw/git).

If you want descriptions, we could add them in the table and extract them.

**Comment by kiesman99** [comment]
> The [original Version](https://github.com/zimfw/git/blob/master/functions/git-alias-lookup) includes the descriptions provided in the [README](https://github.com/zimfw/git).
> 
> If you want descriptions, we could add them in the table and extract them.

@joke Here an example:

```bash
❯ G? amend
Gcf  git commit --amend --reuse-message HEAD                          amends the tip of the current branch reusing the same log message as *HEAD*.
GcF  git commit --verbose --amend                                     amends the tip of the current branch.
GrS  git rebase --exec "git commit --amend --no-edit --no-verify -S"  rebases and GPG-signs the commits.
```

**Review by joke — APPROVED** [review]

**Pull request merged**